src/Microsoft.DotNet.Interactive.SqlServer/MsSqlKernel.cs

```diff
@@ -20,7 +20,7 @@ internal class MsSqlKernel : ToolsServiceKernel
         internal MsSqlKernel(
             string name,
             string connectionString,
-            ToolsServiceClient client) : base(name, client, "SQLite")
+            ToolsServiceClient client) : base(name, client, "SQL")
```

Comment:
`T-SQL`?

Reply:
@brettfo  what are you thoughts on this language

Reply:
"SQL" is good; it'll play well with my kernel picker work